icd/api/vk_cmdbuffer.cpp

```diff
@@ -1817,13 +1817,13 @@ PFN_vkCmdBindDescriptorSets CmdBuffer::GetCmdBindDescriptorSetsFunc(
 {
     PFN_vkCmdBindDescriptorSets pFunc = nullptr;
 
-    if (pDevice->GetEnabledFeatures().robustBufferAccess)
+    if (pDevice->GetRuntimeSettings().enableRelocatableShaders || pDevice->GetEnabledFeatures().robustBufferAccess)
     {
-        pFunc = CmdBindDescriptorSets<numPalDevices, true>;
+        pFunc = CmdBindDescriptorSets<numPalDevices, /* useCompactDescriptor = */ false>;
```

Comment:
Our convention is to not have comments in the parameter list.

Reply:
done